### ISSUE TITLE
🐞 BUG Global styles (v3.0+) not working on theme generated pages

### ISSUE THREAD

**Issue by perks98** [elementor/elementor#12637]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
Hi,
There is an issue with the use of the latest global styles feature, on theme or plugin generated pages like blog pages and product pages global styles are not working as it is missing the elementor kit class.

The section on a standard page edited with Elementor:
![Capture48](https://user-images.githubusercontent.com/55829113/94033907-52e8ff80-fdb9-11ea-890d-6123a7a90443.PNG)

The same section saved as a template and placed in a sidebar on the blog page:
![Capture49](https://user-images.githubusercontent.com/55829113/94033902-52506900-fdb9-11ea-8c85-4915ecce3723.PNG)

## Steps to reproduce
You can reproduce this by creating a section template with global colours applicated to a widget within then add this template to the blog sidebar by using Elementor Library widget, save, look at the front end and you will notice that the 'elementor-kit-[number]' class in the body tag is missing which applies the global styles.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by lior-sh** [comment]
Hi @perks98,

Thanks for reporting, we couldn't reproduce the issue.
Please upgrade to the latest version 3.0.10 and check if the issue reproduced.

Thanks.

**Comment by perks98** [comment]
Hi @lior-sh still getting the issue in the latest free and pro update, I've cleared all cache and disabled all plugins except Elementor's. I've tried it on a few themes and still get the issue where elementor kit class isn't in the body tag on theme pages like the theme blog page, but if I custom made the blog page with Elementor then it would work, just not the theme pages. Once I use the inspector tool with Chrome and add the class manually the styling is applied. This was on a test sub-site from our multisite as most of the sites I make is done on that. I've tried it on Astra and Twenty Twenty themes.

**Comment by lior-sh** [comment]
Hi @perks98,

Can you please write the full steps to replicate & the system info?
And please attach a video that shows the issue, you can use the Loom chrome extension for that.
Thanks.

**Comment by perks98** [comment]
@lior-sh 
Will this do?
https://www.loom.com/share/b9db1048fe7f4bb787df04792eba0d40

<details>
<summary>System Info</summary>
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v5.7.31
	PHP Version: 7.3.21
	PHP Max Input Vars: 2500
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5.1
	Site URL: https://brainstorm-access.elementorwp.uk
	Home URL: http://brainstorm-access.elementorwp.uk
	WP Multisite: Yes
	Max Upload Size: 1 MB
	Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-GB
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Twenty Twenty
	Version: 1.5
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.121 Safari/537.36

== Active Plugins ==
	Elementor Pro
		Version: 3.0.5
		Author: Elementor.com


== Network Plugins ==
	Elementor
		Version: 3.0.10
		Author: Elementor.com


== Must-Use Plugins ==
	StackCache
		Version: 
		Author: Stack CP


== Elements Usage ==
	
	page : 1
		button : 2
		heading : 1
		template : 1
	section : 2
		button : 4
		heading : 1
	wp-post : 0
		button : 2
		divider : 1
		heading : 7
		theme-page-title : 1
	wp-page : 1
		button : 2


== Log ==
	
Log: showing 3 of 32020-09-22 08:09:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.2',
  'to' => '3.0.4',
)]
2020-09-24 07:45:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.9',
  'to' => '3.0.10',
)]
2020-09-24 07:56:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.4',
  'to' => '3.0.5',
)]

PHP: showing 1 of 1PHP: 2020-09-22 08:23:36 [notice X 2][/home/sites/7a/3/3e1894e553/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


</details>

**Comment by lior-sh** [comment]
Hi @perks98,

Thanks for reporting.
We will address this issue soon and release a patch.
Please follow this thread for updates.
Thanks!

**Comment by arielk** [comment]
This issue has been resolved in Elementor v3.0.11

Feel free to update

Thanks!
